client/components/domains/contact-details-form-fields/index.jsx

```diff
@@ -127,6 +128,16 @@ export class ContactDetailsFormFields extends Component {
 		this.inputRefCallbacks = {};
 		this.formStateController = null;
 		this.shouldAutoFocusAddressField = false;
+
+		// Temporary: This should be removed after we're
+		// certain no existing code constructs an object with.
+		// null onSubmit. See wp-calypso PR #38835
+		if ( ! this.props.onSubmit ) {
```

Comment:
I didn't see the feedback that inspired this change, but IMO there's no need to worry or log; if there's no `onSubmit` prop, the button can't do anything and would throw an error when it got to `this.props.onSubmit( this.getMainFieldValues() )`, so if this breaks something, that thing is already broken.